src/settings.js

```diff
@@ -20,6 +20,7 @@ function expandObjBy(obj1, obj2) {
 }
 
 function trim(obj1, obj2) {
+	if (obj1 instanceof Array) { return; }
```

Comment:
I think `Array.isArray(obj1)` is a better check and it's what we use everywhere else to check if a variable is an array.